src/main/resources/banner.txt

```diff
@@ -0,0 +1,6 @@
+================================================================================
+
```

Comment:
What is this? Doesn't look used, can delete it. If you want to keep the directory - add a blank file